pkg/location/location.go

```diff
@@ -97,7 +98,12 @@ func writeData(ctx context.Context, pType objectstore.ProviderType, profile para
 	if err != nil {
 		return err
 	}
-	if err := bucket.Put(ctx, path, in, 0, nil); err != nil {
+	b, err := ioutil.ReadAll(in)
```

Comment:
Won't we end up reading the data twice?

Reply:
yeah, we discussed this in yesterday's call. We unknowingly introduced another issue here. After reading the `in` is going to be `nil`. I am working on this.